### PR TITLE
Database ui fix

### PR DESCRIPTION
**Pull request by w2ji**

### CODE REVIEW

**Comment by divad12** [comment]
Cool, what does this do?

File: src/com/uwflow/flow_android/custom_view/DragZoomImageView.java [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+import android.graphics.Bitmap;
+import android.graphics.Matrix;
+import android.graphics.Point;
+import android.graphics.PointF;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.ImageView;
+
+public class DragZoomImageView extends ImageView implements View.OnTouchListener {
```
Comment (divad12) [inline src/com/uwflow/flow_android/custom_view/DragZoomImageView.java:13]: Did you write all this code, or is much of this obtained from somewhere?

If the latter, do you mind linking to where you got this from?

Comment (w2ji) [inline src/com/uwflow/flow_android/custom_view/DragZoomImageView.java:13]: okay leme see if I can find the link

Comment (w2ji) [inline src/com/uwflow/flow_android/custom_view/DragZoomImageView.java:13]: http://stackoverflow.com/questions/6650398/android-imageview-zoom-in-and-zoom-out

File: src/com/uwflow/flow_android/fragment/ProfileFragment.java [hunk 2]
```diff
@@ -39,74 +41,221 @@
...
+//                                                    @Override
+//                                                    public void onSuccess() {
+//                                                        userCover = ImageHelper.drawableToBitmap(mCoverPhoto.getDrawable());
+//                                                    }
+//
+//                                                    @Override
+//                                                    public void onError() {
+//
+//                                                    }
+//                                                });
```
Comment (divad12) [inline src/com/uwflow/flow_android/fragment/ProfileFragment.java:157]: Can we remove this commented-out code?

Comment (w2ji) [inline src/com/uwflow/flow_android/fragment/ProfileFragment.java:157]: yep

**Pull request merged**